index.js

```diff
@@ -1,238 +1,3 @@
-const get = require("lodash/get");
-const isEqual = require("lodash/isEqual");
-const isEmpty = require("lodash/isEmpty");
-const isNil = require("lodash/isNil");
-const set = require("lodash/set");
+var formLinker = require("./dist");
```

Comment:
I believe the main index.js file will need to reference the transpiled file in the dist folder for the babel transforms to be applied  when formLinker is imported as a dependency. For that reason I moved the source code to a src/index.js location. The remaining changes are simply to keep testing, linting, and building working based on this new file location. 

If you don't like this choice, I'll be happy to change things according to your preference - or feel free to make the changes yourself if you prefer.

Reply:
@EvanMorrison can we talk about why this works when you get back.